modules/exploits/windows/local/bits_ntlm_token_impersonation.rb

```diff
@@ -206,8 +206,7 @@ def check
   # WinRM is running, it returns 2. And if both are running, it returns
   # BITS + WINRM = 3.
   def check_bits_and_winrm
-    check_command = 'powershell.exe Get-Service -Name BITS,WinRM'
-    result = cmd_exec(check_command)
+    result = cmd_exec('cmd.exe', '/c echo . | powershell.exe Get-Service -Name BITS,WinRM')
```

Comment:
I believe this could be updated to use `sc query WinRM` and `sc query BITS` and remove the need for powershell

Reply:
This module only supports Meterpreter not shell or PowerShell sessions. Why not use `service_status` from `lib/msf/core/post/windows/services.rb` ?

Reply:
Good call; In this case I'm just here to fix the crash and didn't want to scope creep/rewrite/retest things